x-pack/plugins/cases/server/services/notifications/email_notification_service.ts

```diff
@@ -95,16 +92,15 @@ export class EmailNotificationService implements NotificationService {
         .map((user) => user.email);
 
       const subject = EmailNotificationService.getTitle(theCase);
-      const message = EmailNotificationService.getMessage(
-        theCase,
-        this.spaceId,
-        this.publicBaseUrl
-      );
+      const message = EmailNotificationService.getPlainTextMessage(theCase, caseUrl);
+
+      const messageHTML = await getEmailBodyContent(theCase, caseUrl);
```

Comment:
Actually that's not going to help because the `EmailNotificationService` object is initialized with each request. We'd need some more complicated. Maybe that's not worth doing for this release 🤷‍♂️ what do you think @cnasikas ?